experiments/collect_data.py

```diff
@@ -99,7 +99,7 @@ def main(
     "acdc": TASKS,
     "16h": TASKS,
     "sp": TASKS,
-    "canonical": ["ioi", "greaterthan"],
```

Comment:
Am I right in thinking that you're keeping the `tasks_for["canonical"]` equal to the list of canonical circuits we've correctly implemented so far?

PR looks good!